webservice/webservice.py

```diff
@@ -322,7 +329,79 @@ def get_manifest():
     # Get the response back
     logger.info("Creating the API response")
     response = es_td.transform_manifest(filters=filters)
-    # Return the excel file
+    if format == 'tsv':  # returns metadata as TSV file
+        return response
+    elif format == 'bdbag':  # returns presigned URL to a BDBag in S3
+        response_bag = bdbag_response(response)
+        return jsonify(response_bag)
+
+
+def bdbag_response(response_obj):
+    """
+    Create a response for BDBag upload to S3. Returns just the presigned URL of
+    the S3 location if all goes well. Otherwise the HTTP status code. This that
+    can go wrong include 
+    :param response_obj: Contains the selected metadata as a TSV.
+    :type response_obj: A Flask response object.
+    :return response: Information depends on status of actions.
+    :rtype response: JSON
+    """
+    logger = logging.getLogger('dashboardService.webservice.bdbag_response')
+    # Create and return the BDbag folder.
+    bag_name = 'manifest'
+    bag_info = {'organization': '',
+                'data_type': '',
+                'date_created': datetime.datetime.now().isoformat()}
+
+    # Instantiate bag object.
+    bag = BagHandler(data=response_obj.get_data(),
+                     bag_info=bag_info,
+                     bag_name=bag_name)
+    zipped_bag = bag.create_bag()  # path to compressed bag
+    logger.info("Creating a compressed BDbag containing manifest.")
+
+    # Import bucket environment variable, launch instance of S3-file handler,
+    # and upload the BDBag file to S3.
+
+    # Transfer parameters.
+    aws_region = 'us-west-2'
```

Comment:
You are not using the parameter. This should not be hard-coded.

Reply:
Not sure how to fix this. In my instance, AWS_REGION isn't an environment variable. I suggest we complete this PR, merge with feature/commons, then read out the region here, and redeploy the instance.

Reply:
This is still not fixed.